core/server/services/settings/ensure-settings.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const fs = require('fs-extra'),
+    path = require('path'),
+    debug = require('ghost-ignition').debug('settings:ensure-settings'),
+    common = require('../../lib/common'),
+    knownSettings = require('./index').knownSettings,
+    config = require('../../config');
+
+/**
+ * Makes sure that all supported settings files are in the
+ * `/content/settings` directory. If not, copy the default files
+ * over.
+ * @returns {Promise}
+ * @description Reads the `/settings` folder of the content path and makes
+ * sure that the associated yaml file for each setting exists. If it doesn't
+ * opy the default yaml file over.
+ */
+module.exports = function ensureSettingsFiles() {
+    const contentPath = config.getContentPath('settings'),
+        defaultSettingsPath = config.get('paths').defaultSettings;
+
+    return Promise.each(knownSettings, function (setting) {
```

Comment:
I know we override the global `Promise`, but my IDE complains that `Promise` is not imported. This is for sure configureable, but didn't research yet. My IDE is already configured to ES6.
And i would say, either we import it nowhere or everywhere.

Reply:
> My IDE is already configured to ES6.

I don't know how I could change that in Atom. I'm using the project `.eslintrc` which is set to ES6 as well and doesn't complain for me 🤔 

> And i would say, either we import it nowhere or everywhere.

This would be more of a general thing? Or what exactly do you mean?

Reply:
> I'm using the project .eslintrc which is set to ES6

Same. But complains still.

> And i would say, either we import it nowhere or everywhere.

Yeah this would be a different task. For now i would say, we stay consistent?